mix.exs

```diff
@@ -121,11 +122,12 @@ defmodule EventStore.Mixfile do
 
   defp package do
     [
-      files: ["lib", "priv/event_store", "guides", "mix.exs", "README*", "LICENSE*"],
+      files: ["lib", "priv/event_store", "guides", "mix.exs", "README*", "LICENSE*", "CHANGELOG*"],
       maintainers: ["Ben Smith"],
       licenses: ["MIT"],
       links: %{
-        "GitHub" => "https://github.com/commanded/eventstore"
+        "Changelog" => "#{@source_url}/blob/master/CHANGELOG.md",
```

Comment:
The CHANGELOG is also published to Hex so it's possible to link to the exact release version:

```suggestion
        "Changelog" => "https://hexdocs.pm/eventstore/{@version}/changelog.html",
```